docs/reference/migration/migrate_8_0/settings.asciidoc

```diff
@@ -185,3 +185,27 @@ If you would like to use wildcard patterns for destructive actions, set
 `action.destructive_requires_name` to `false` using the <<cluster-update-settings,
 cluster settings>> API.
 ====
+
+.Legacy role settings are removed
+[%collapsible]
+====
+*Details* +
+The legacy role settings:
+
+* `node.data`
+* `node.ingest`
+* `node.master`
+* `node.ml`
+* `node.remote_cluster_client`
+* `node.transform`
+* `node.voting_only`
+
+have been removed. Instead, use the `node.roles` setting. If you were previously
+using the legacy role settings on 7.13 or later cluster, you will have a
+deprecation log message on each of your node informing you of the exact
```

Comment:
Maybe reword the last part of this:

on each of your **nodes indicating the exact replacement value for `node.roles`.**

Reply:
I pushed 2362e7cb13f88272373965c757d62c17cbed8b88.